gobcore/model/amschema/repo.py

```diff
@@ -9,8 +9,8 @@
 from gobcore.model.amschema.model import Dataset, Table
 from gobcore.parse import json_to_cached_dict
 
-REPO_BASE = os.getenv(
-    'REPO_BASE', "https://raw.githubusercontent.com/Amsterdam/amsterdam-schema/master")
+
+REPO_BASE = os.getenv("REPO_BASE", "https://raw.githubusercontent.com/Amsterdam/amsterdam-schema/master")
```

Comment:
Wat is de regellengte die we nu hanteren?

Reply:
119, but is not forced yet